build.gradle

```diff
@@ -41,6 +41,11 @@ ext {
 
 wpilibTools.deps.wpilibVersion = wpilibVersion
 
+// Tell gradlerio what version of things to use (that we care about)
+// See: https://github.com/wpilibsuite/GradleRIO/blob/main/src/main/java/edu/wpi/first/gradlerio/wpi/WPIVersionsExtension.java
+wpi.getVersions().getOpencvVersion().convention(openCVversion);
+wpi.getVersions().getWpilibVersion().convention(wpilibVersion);
+
```

Comment:
Pretty sure this is handled by [config.gradle](https://github.com/PhotonVision/photonvision/blob/6666b22fc1ebc19503565f40c598960382d9da52/shared/config.gradle#L13) already

Reply:
Nope. Config.gradle is not included outside of photonlib

Reply:
Also that's for native utils, not gradle Rio